OloEngine/src/OloEngine/Asset/SoundGraphAsset.cpp

```
@@ -101,8 +101,16 @@ namespace OloEngine
     {
         OLO_PROFILE_FUNCTION();
 
-        // Validate nodes exist
-        if (HasNode(connection.m_SourceNodeID) && HasNode(connection.m_TargetNodeID))
+        // Endpoint must either resolve to a real node, or be the graph-IO sentinel
+        // (UUID(0) on the source side = "from graph input", UUID(0) on the target side =
+        // "to graph output"). Rejecting UUID(0) here would diverge from the serializer
+        // path — which simply appends to m_Connections — and silently drop every
```

Cppcheck (2.20.0)

[style] The function 'GetGraphInputs' is never used.

(unusedFunction)